compiler/luci/pass/src/helpers/Strings.h

```diff
@@ -36,6 +36,10 @@ std::string to_lower_case(std::string);
 
 loco::DataType str_to_dtype(const std::string &);
 
+std::vector<std::string> parse_comma_separated_str(const std::string &);
```

Comment:
We could use existing library.

https://github.com/Samsung/ONE/blob/9ff5081c91f4fd9155f38beda649d0424a8b5a7a/compiler/pepper-csv2vec/include/pepper/csv2vec.h#L26

Reply:
Thanks for notice. I'll update source code to reduce duplicate code.

I'm a bit surprised that there is an individual module just for this purpose.